Sources/Segment/Configuration.swift

```diff
@@ -95,5 +99,16 @@ public extension Configuration {
         values.cdnHost = value
         return self
     }
+    
+    @discardableResult
+    func errorHandler(_ value: @escaping (Error) -> Void) -> Configuration {
```

Comment:
Just curious as to why it returns the `Configuration` ?

Reply:
The config methods are all chainable and this continues that.  Makes is so you can do ...

```
let analytics = Configuration(writeKey: "1234")
                         .trackLifecycleEvents(true)
                         .apiHost("eu1.segmentapi.com")
                         .whateverElse()
```

.. instead of having to create an instance and then set them all afterwards.